src/main/java/rs/teslaris/core/service/interfaces/commontypes/ResearchAreaService.java

```diff
@@ -14,7 +11,7 @@ public interface ResearchAreaService extends JPAService<ResearchArea> {
 
     ResearchArea getReferenceToResearchAreaById(Integer id);
 
-    Page<ResearchAreaResponseDTO> getResearchAreas(Pageable pageable);
+    List<rs.teslaris.core.dto.commontypes.ResearchAreaDTO> getResearchAreas();
```

Comment:
Why this is the part of this PR? It is not connected to upgrade of spring boot, right?

Reply:
This is a Bugfix for the functionality I was working on... not related to migration.